src/containers/settings/actions/index.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { newRequestSequence } from 'redux-reqseq';
+import type { RequestSequence } from 'redux-reqseq';
+
+const GET_APP_SETTINGS :'GET_APP_SETTINGS' = 'GET_APP_SETTINGS';
+const getAppSettings :RequestSequence = newRequestSequence(GET_APP_SETTINGS);
+
+const UPDATE_APP_SETTINGS :'UPDATE_APP_SETTINGS' = 'UPDATE_APP_SETTINGS';
+const updateAppSettings :RequestSequence = newRequestSequence(UPDATE_APP_SETTINGS);
+
+const CREATE_APP_SETTINGS :'CREATE_APP_SETTINGS' = 'CREATE_APP_SETTINGS';
+const createAppSettings :RequestSequence = newRequestSequence(CREATE_APP_SETTINGS);
```

Comment:
Is create called anywhere?

Reply:
not yet. this feature currently doesn't do anything for orgs that don't already have some app settings initialized. that bit is planned for later. but since we aren't actively onboarding at the moment, i've chosen to save that work for later. this feature is mainly for us to manage existing orgs.